hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/container/replication/TestRatisOverReplicationHandler.java

```diff
@@ -236,6 +237,46 @@ public void testOverReplicatedContainerBecomesMisReplicatedOnRemoving()
         getOverReplicatedHealthResult(), 0);
   }
 
+  @Test
+  public void testOverReplicatedAllUnhealthyPicksLowestBCSID()
```

Comment:
Thanks for the patch @sodonnel. Should this test be named `testOverReplicatedAllUnhealthySameBCSID` and the test at L260 be named `testOverReplicatedAllUnhealthyPicksLowestBCSID`

Reply:
Ah, yes you are correct. I guess I copied the tests and then edited the wrong one! I have fixed that.